types/jasmine/ts3.1/index.d.ts

```diff
@@ -155,6 +156,18 @@ interface DoneFn extends Function {
     fail: (message?: Error | string) => void;
 }
 
+/**
+ * Configuration that can be used when configuring Jasmine via {@link jasmine.Env.configure}
+ */
+interface Configuration {
+    random?: boolean;
+    seed?: number;
+    failFast?: boolean;
+    oneFailurePerSpec?: boolean;
+    hideDisabled?: boolean;
+    specFilter?: Function;
```

Comment:
Does it make sense to use `Fn` instead?

Reply:
Not sure, tbh

Reply:
We use this type each time we expect function, so let's use it here as well. Same for previous file.